moVirt/src/main/java/org/ovirt/mobile/movirt/ui/triggers/BaseTriggerActivity.java

```diff
@@ -178,4 +182,19 @@ private int asIntWithDefault(String toParse, String defaultResult) {
     public void homeSelected() {
         onBackPressed();
     }
+
+    @Receiver(actions = {Broadcasts.ERROR_MESSAGE},
+            registerAt = Receiver.RegisterAt.OnResumeOnPause)
+    public void showErrorDialog(
+            @Receiver.Extra(Broadcasts.Extras.ERROR_REASON) String reason,
+            @Receiver.Extra(Broadcasts.Extras.REPEATED_MINOR_ERROR) boolean repeatedMinorError) {
+        CreateDialogBroadcastReceiverHelper.showErrorDialog(getFragmentManager(), reason, repeatedMinorError);
+    }
+
+    @Receiver(actions = {Broadcasts.REST_CA_FAILURE},
+            registerAt = Receiver.RegisterAt.OnResumeOnPause)
+    public void showCertificateDialog(
+            @Receiver.Extra(Broadcasts.Extras.ERROR_REASON) String reason) {
+        CreateDialogBroadcastReceiverHelper.showCertificateDialog(getFragmentManager(), reason, true);
+    }
```

Comment:
I don't know any better way, how to put this common functionality into all of our activities